docs/release-notes/index.md

```diff
@@ -23,6 +23,10 @@ To check for security updates, go to [Security announcements for the Elastic sta
 
 ## 9.0.1 [logstash-9.0.1-release-notes]
 
+::::{important}
+The 9.0.1 release contains fixes for potential security vulnerabilities. See our [security advisory](https://discuss.elastic.co/c/announcements/security-announcements/31) for more details.
```

Comment:
```suggestion
The 9.0.1 release contains fixes for potential security vulnerabilities. 
Check out the [security advisory](https://discuss.elastic.co/c/announcements/security-announcements/31) for details.
```

Reply:
Minor tweaks to use "one sentence per line" best practice, and more inclusive language.